app/src/main/java/de/othr/plantico/ui/search/SearchViewModel.kt

```diff
@@ -0,0 +1,11 @@
+package de.othr.plantico.ui.search
+
+import androidx.lifecycle.ViewModel
+import de.othr.plantico.database.entities.Plant
+
```

Comment:
This ViewModel does not make sense, a ViewModel should hold the data itself. If you look at PlantViewModel, the ViewModel is created by a ViewModelFactory which passes the database repository, and the ViewModel holds the plant lists. If the data is passed by the PlantViewModel anyway, I don't see the reason for another ViewModel.

Reply:
Deleted the ViewModel and put the search function to the PlantViewModel as discussed.
Please have a look again :)